tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Jasmine/js/angulartics-azure.js

```diff
@@ -0,0 +1,47 @@
+﻿/**
```

Comment:
we don't need angularatics in test project unless you are mocking it and injecting for testing.